guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,8 +4,39 @@
 [id="foreman-headline-features"]
 == Headline Features
 
+=== Enabled HTTP/2 on Apache
+
+HTTP/2 is now enabled by default on all platforms.
+Compared to HTTP/1(.1) the newer HTTP/2 protocol allows multiplexing over a single connection, making it possible for clients to retrieve multiple resources at the same time.
+For example, browsers no longer need to wait for one one image to complete before requesting the next image.
+
+=== Use of system crypto policy with Apache on Enterprise Linux 8
+
+Red Hat Enterprise Linux 8 https://www.redhat.com/en/blog/consistent-security-crypto-policies-red-hat-enterprise-linux-8[introduced consistent crypto policies] and Foreman's installer now configures Apache to use those.
+Where previously the installer manually maintained a list of protocols and ciphers, it now follows what is configured system wide.
+
+Note that Katello users can't use the FUTURE policy since `cdn.redhat.com` has a cerificate signed with SHA1 in its signature chain.
+This was already a problem prior to this change.
+See https://bugzilla.redhat.com/show_bug.cgi?id=2117265[Red Hat BZ#2117265] for more information.
+
+For Debian/Ubuntu users nothing has changed.
+
+=== Redis 6 on Enterprise Linux 8
+
+The installer now ensures Redis 6 is used on Enterprise Linux 8.
+The Redis 5 AppStream https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle[went EOL in May 2022].
+Previously it was up to the user to switch streams, but now the installer enforces the `redis:6` stream and updates the package.
+
+=== Allow manual modifications of ansible.cfg
+
+The installer no longer manages ansible.cfg and thus allows users to put in their own modifications.
+This file was previously used to configure the options that Foreman required to have set.
+This is no longer the case as the necessary options are set using a different mechanism.
+During upgrades the file is not touched which means some cruft remains.
+It is safe to remove the file or remove all content.
+
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
-[id="foreman-deprecations"]
-== Deprecations
+* The `foreman_chef` plugin is unmaintained and broken since a few releases.
```

Comment:
@ekohl Does this mean chef is officially no longer supported once 3.5 is stable?

Reply:
Correct - it's broken (as in: it didn't even install anymore) and without a maintainer it was decided to retire the packages and drop it from the installer.

I welcome any improvements on the release notes. I merged it quickly so I could release RC1 but before RC2 and GA it deserves more attention.